README.md

```diff
@@ -31,6 +31,7 @@ exceptions/differences/extensions (:white_check_mark: are the implemented sniffs
 - :white_check_mark: Omit phpDoc for parameters/returns with native types, unless adding description
 - :white_check_mark: Don't use `@author`, `@since` and similar annotations that duplicate Git information
 - :white_check_mark: Assignment in condition is not allowed
+- :white_check_mark: Use early return instead of `else`
```

Comment:
The comment is not true. The sniff does a little more :)

Reply:
Have a suggestion? :smile: